packages/sane_backends.rb

```diff
@@ -3,33 +3,34 @@
 class Sane_backends < Package
   description 'Scanner Access Now Easy - Backends'
   homepage 'http://www.sane-project.org/'
-  version '1.0.27'
+  version '1.0.32'
   compatibility 'all'
-  source_url 'https://gitlab.com/sane-project/backends/-/archive/RELEASE_1_0_27/backends-RELEASE_1_0_27.tar.bz2'
-  source_sha256 '6a4034a9d29255bb40cf5a19f3d85dd7ba4fe6452eb9b1ae4385bc8175493ec4'
+  source_url "https://gitlab.com/sane-project/backends/-/archive/#{version}/backends-#{version}.tar.bz2"
+  source_sha256 'c0425aad44e4f79dbf2f123d3060d690c5c328ef58316e0ad0322c61d63a4783'
```

Comment:
Did you generate new binaries with this new release?

Reply:
Not once again once I changed the URL a second time. I can do so if needed...

Reply:
Only the old archive works to build. not the new one. So reverting the URL change.